data/data/bootstrap/baremetal/systemd/units/extract-machine-os.service

```diff
@@ -7,6 +7,7 @@ After=crio.service build-ironic-env.service ironic-volume.service
 # The rootfs file must be created before the httpd container is started
 # otherwise the kernel params in inspector.ipxe will not be set correctly
 Before=ironic-httpd.service
+OnSuccess=image-customization.service
```

Comment:
Slightly surprising that this works because `RemainAfterExit=true` means "the service shall be considered active even when all its processes exited", but I guess it does.

This is potentially hard to maintain because if we ever in future added another service that depends on this, we could very easily forget to add that one to this list. But I guess we have to do what works.